### PR TITLE
feat: Display qualified users in federated conversation

### PR DESCRIPTION
**Pull request by ffflorian**

### CODE REVIEW

**Review by Yserz — APPROVED** [review]

File: src/script/entity/Conversation.ts [hunk 1]
```diff
@@ -126,8 +127,12 @@ export class Conversation {
   public readonly messageTimer: ko.PureComputed<number>;
   public readonly name: ko.Observable<string>;
   public readonly notificationState: ko.PureComputed<number>;
+  /** Does not include self user */
   public readonly participating_user_ets: ko.ObservableArray<User>;
+  /** Does not include self user */
   public readonly participating_user_ids: ko.ObservableArray<string>;
+  /** Does not include self user */
```
Comment (bennycode) [inline src/script/entity/Conversation.ts]:
```suggestion
  /** Contains only IDs from users with domains. Does not include self user. */
```

File: src/script/entity/Conversation.ts [hunk 2]
```diff
@@ -126,8 +127,12 @@ export class Conversation {
   public readonly messageTimer: ko.PureComputed<number>;
   public readonly name: ko.Observable<string>;
   public readonly notificationState: ko.PureComputed<number>;
+  /** Does not include self user */
   public readonly participating_user_ets: ko.ObservableArray<User>;
+  /** Does not include self user */
```
Comment (bennycode) [inline src/script/entity/Conversation.ts]:
```suggestion
  /** Contains only IDs from users without a domain. Does not include self user. */
```

File: src/script/entity/Conversation.ts [hunk 3]
```diff
@@ -126,8 +127,12 @@ export class Conversation {
   public readonly messageTimer: ko.PureComputed<number>;
   public readonly name: ko.Observable<string>;
   public readonly notificationState: ko.PureComputed<number>;
+  /** Does not include self user */
```
Comment (bennycode) [inline src/script/entity/Conversation.ts]:
```suggestion
  /** Includes all participating users (with & without domain) except the self user. */
```

File: src/script/entity/Conversation.ts [hunk 4]
```diff
@@ -454,7 +464,7 @@ export class Conversation {
           return truncate(joinedNames, maxLength, false);
         }
 
-        const hasUserIds = !!this.participating_user_ids().length;
+        const hasUserIds = this.participating_user_ids().length + this.participatingQualifiedUserIds().length > 0;
```
Comment (bennycode) [inline src/script/entity/Conversation.ts]: How about exposing this value (`this.participating_user_ids().length + this.participatingQualifiedUserIds().length`) in a getter for `Conversation.ts`?

**Review by bennycode — CHANGES_REQUESTED** [review]

**Review by bennycode — APPROVED** [review]

**Pull request merged**